src/main/cpp/feedback/DriverFeedback.cpp

```diff
@@ -169,13 +175,32 @@ void DriverFeedback::UpdateDiagnosticLEDs()
             algaeSensor = taleMgr->GetAlgaeSensorState();
         }
     }
-    m_LEDStates->DiagnosticPattern(FMSData::GetInstance()->GetAllianceColor(), coralInSensor, coralOutSensor, algaeSensor, m_questStatus, m_ll1Status);
-    if (m_timer == 20)
+    auto chassisConfig = ChassisConfigMgr::GetInstance()->GetCurrentConfig()->GetSwerveChassis();
+    if (chassisConfig != nullptr)
     {
-        QueryNT();
-        m_timer = 0;
+        auto visionPoseEstitmators = chassisConfig->GetSwervePoseEstimator()->GetVisionPoseEstimators();
+        if (visionPoseEstitmators.size() > 0)
```

Comment:
it is preferred to do the following instead of size() > 0

if (!visionPoseEstimators.empty())